Tests/LibWeb/Text/input/link-changed-rel.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<script src="include.js"></script>
+<script>
+    asyncTest(done => {
+        let link = document.createElement("link");
+        link.setAttribute("href", "valid.css");
+        link.addEventListener("load", function() {
+            println("Got load event");
+        });
+        document.head.appendChild(link);
+
+        // link becomes a stylesheet (event)
+        link.setAttribute("rel", "stylesheet");
+
+        // link is still a stylesheet (no event)
+        link.setAttribute("rel", "preconnect stylesheet");
+
+        setTimeout(done, 1000);
+    });
+</script>
```

Comment:
Running this test on the `master` branch gives the same result as running it on the PR branch for me. I think you'd need to do `link.setAttribute` inside the event handler to see a difference.

Reply:
oops, should've checked :sweat_smile: